features/include/pcl/features/impl/feature.hpp

```diff
@@ -203,12 +203,12 @@ Feature<PointInT, PointOutT>::compute (PointCloudOut &output)
   output.header = input_->header;
 
   // Resize the output dataset
-  if (output.points.size () != indices_->size ())
+  if (output.size () != indices_->size ())
     output.points.resize (indices_->size ());
 
   // Check if the output will be computed for all points or only a subset
   // If the input width or height are not set, set output width as size
-  if (indices_->size () != input_->points.size () || input_->width * input_->height == 0)
+  if (indices_->size () != input_->size () || input_->width * input_->height == 0)
   {
     output.width = static_cast<std::uint32_t> (indices_->size ());
```

Comment:
remove cast

Reply:
`static_cast<decltype(output.width)>`? Else clang most likely will complain

Reply:
I'm ok with that. That being said, I thought clang only complained on signed/unsigned comparisons. Removal should not be an issue https://godbolt.org/z/yB4tR9 (I enabled -Wall and -Wextra for both gcc and clang)

Reply:
Yup. Only MSVC warns on assignment sign mismatch